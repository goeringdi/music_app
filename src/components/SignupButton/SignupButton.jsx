import {Button} from '../../pages/Signup/styles'

export const SignupButton = () => {
  return <Button>Зарегистрироваться</Button>
}
