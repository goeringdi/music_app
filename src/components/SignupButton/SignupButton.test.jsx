import { SignupButton } from "./SignupButton";
import { screen, render } from "@testing-library/react";


describe("<Button />", () => {
  describe("Snapshots tests", () => {
  it("should render successfully and match snapshot", () => {
    const { container } = render(<SignupButton>Зарегистрироваться</SignupButton >);

    expect(container).toMatchSnapshot();
  });
  })

  describe("Attributes tests", () => {
    it('should set type="button" by default', () => {
      render(<SignupButton />);

      expect(screen.getByRole("button")).toHaveAttribute("type", "button");
    });
  });
})