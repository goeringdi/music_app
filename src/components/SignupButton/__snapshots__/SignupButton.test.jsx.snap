// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Snapshots tests should render successfully and match snapshot 1`] = `
<div>
  <button
    class="sc-eDvSVe phOoF"
  >
    Зарегистрироваться
  </button>
</div>
`;


